pycryptobot.py

```diff
@@ -1820,7 +1825,10 @@ def runApp(_websocket):
                 f"{str(datetime.now())} bot is closed via keyboard interrupt..."
             )
         try:
-            telegram_bot.removeactivebot()
+            try:
+                telegram_bot.removeactivebot()
+            except:
+                pass
```

Comment:
silently swallowing errors without any reporting or writing to the log is not really recommended